packages/neuron-wallet/src/models/system-script-info.ts

```diff
@@ -35,8 +38,16 @@ export default class SystemScriptInfo {
 
   private daoOutPointInfo = new Map<string, OutPoint>()
 
+  private legacyMultiSignOutPointInfo = new Map<string, OutPoint>()
   private multiSignOutPointInfo = new Map<string, OutPoint>()
 
+  public static getMultiSignHashType(lockCodeHash: string): ScriptHashType {
+    if (lockCodeHash === SystemScriptInfo.MULTISIG_CODE_HASH) {
+      return SystemScriptInfo.MULTI_SIGN_HASH_TYPE
+    }
+    return SystemScriptInfo.LEGACY_MULTI_SIGN_HASH_TYPE
```

Comment:
It should be stricter as follows
```js
if (lockCodeHash === SystemScriptInfo.MULTISIG_CODE_HASH) return SystemScriptInfo.MULTI_SIGN_HASH_TYPE
if (lockCodeHash === SystemScriptInfo.LEGACY_MULTISIG_CODE_HASH) return SystemScriptInfo.LEGACY_MULTI_SIGN_HASH_TYPE
throw new Error("Unknown Multisig Code Hash")
```

Fallback to a value is kind of dangerous when it's about ownership.

BTW, here are 2 spellings of `Multisig` and we can only use one of them.

Reply:
I find `isMultiSignCodeHash` at https://github.com/nervosnetwork/neuron/pull/3347/files#diff-dfaf714e575f95a936d8819381848306d8e8e26b1a065a203f4f5b6c3dddc90eR115 which can be used here to limit the code hash